test/lib/mayaUsd/render/mayaToHydra/testBasicRender.py

```diff
@@ -49,22 +56,27 @@ def test_flat_orange(self):
         cmds.setAttr('persp.rotate', 0, 0, 0, type='float3')
         cmds.setAttr('persp.translate', 0, .25, .7, type='float3')
 
-        self.assertSnapshotEqual("flat_orange.png")
-        self.assertRaises(AssertionError,
-                          self.assertSnapshotEqual, "flat_orange_bad.png")        
+        self.assertSnapshotClose(
+            "flat_orange.png", self.IMAGE_DIFF_FAIL_THRESHOLD,
+            self.IMAGE_DIFF_FAIL_PERCENT)
+        self.assertRaises(
+            AssertionError, self.assertSnapshotClose, "flat_orange_bad.png",
+            self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)
 
-class TestMayaHydraRender(mtohUtils.MayaHydraBaseTestCase):
+class TestMayaHydraRender(BasicRenderBaseTestCase):
     _file = __file__
 
     def test_cube(self):
-        imageVersion = None
-        if maya.mel.eval("defaultShaderName") != "standardSurface1":
-            imageVersion = 'lambertDefaultMaterial'
+        imageVersion = maya.mel.eval("defaultShaderName").rstrip(digits)
```

Comment:
Expected image files are now in a subdirectory based on the default shader name stripped of its numerical suffix, e.g. lambert, standardSurface, openPBRSurface.